src/StreetNameRegistry.Projections.Legacy/StreetNameListV2/StreetNameListProjectionsV2.cs

```diff
@@ -136,6 +136,42 @@ await context
                     }, ct);
             });
 
+            When<Envelope<StreetNameHomonymAdditionsWereCorrected>>(async (context, message, ct) =>
```

Comment:
@ArneD should the event also be handled in StreetNameNameProjectionsV2? This projection doesn't know homonym additions, but maybe `UpdateVersionTimestamp` is necessary?

Reply:
Yes, same in `StreetNameNameProjectionsV2`